src/components/modal/EditModal.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import { Avatar } from "../avatar/Avatar";
+import GifBoxOutlinedIcon from "@mui/icons-material/GifBoxOutlined";
```

Comment:
Suggestion: You can seprate these icons and mui components in separate file and import as your need. It will reduce your importing number or line

Reply:
okay ,Good Idea . Thanks for suggestion. I will do it